modules/library/main/src/main/java/org/geotools/geometry/jts/WKTReader2.java

```diff
@@ -15,13 +15,13 @@
  *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
  *    Lesser General Public License for more details.
  *    
- *    This is a port of the JTS WKTReader to handle SQL MM types such as Curve.
+ *    This is a port of the JTS WKTWRiter to handle SQL MM types such as Curve.
```

Comment:
Small typo - R should be lowercase.

Also this WKTReader2 is described as a port of the JTS WKTWriter, whereas the new WKTWriter2 (below) is described as a port of WKTReader. Is this correct, or should the descriptions be swapped?